DI-1299/README.md

```
@@ -0,0 +1,53 @@
+        ],
+        [
+            "depth_mean", "depth_sd", "gt_depth_mean", "gt_depth_sd",
+            "ab_mean", "ab_std", "p_middling_ab"
+        ]
+    ]
+}
+```
+
+### Running
+Example command:
+```
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)